packages/clay-pagination/src/PaginationWithBasicItems.tsx

```diff
@@ -164,6 +164,11 @@ const ClayPaginationWithBasicItems = React.forwardRef<HTMLUListElement, IProps>(
 					disabled={internalActive === 1}
 					href={previousHref}
 					onClick={() => setActive(previousPage)}
+					role={
+						previousHref || internalActive === 1
```

Comment:
If there is a previousHref, it will be a string which would be a truthy value. Then this would be set as the role, right? So I don't think this logic wouldn't be correct when previousHref exists.

Reply:
If `previousHref` exists, the pagination component outputs `<a class="page-link" href="/url">`. In this case, we don't need `role="button"` because it's a hyperlink.